packager.go

```diff
@@ -61,24 +61,31 @@ func (r *PackageManager) Pack(mediaType MediaType, payload []byte, senderID *cor
 // Unpack returns iden3 message method from envelope
 // if it's not valid or can't be decrypted error is returned
 func (r *PackageManager) Unpack(envelope []byte) (*BasicMessage, MediaType, error) {
-
-	safeEnvelope := strings.Trim(strings.TrimSpace(string(envelope)), "\"")
```

Comment:
safe envelope must be created before determining of MediaType.

Reply:
What reason? We took headers from row envelope